load.Rmd

```diff
@@ -37,7 +37,7 @@ if(!exists("region")) region <- "liverpool-city-region"
 pct_data <- file.path("..", "pct-data")
 pct_bigdata <- file.path("..", "pct-bigdata")
 pct_privatedata <- file.path("..", "pct-privatedata")
-pct_shiny <- file.path("..", "pct-shiny")
+pct_shiny <- file.path("..", "pct-shiny", "regions_www")
```

Comment:
This line (along with this https://github.com/npct/pct-load/pull/32/files#diff-f728b105860d99bed1d5f0718c4ec40bR680 ) will mean we create a `../pct-shiny/regions_www/regions_www` folder.  I think it makes sense to have the `pct_shiny` variable point to the whole repo and not just the `regions_www` folder